doc: add minutes for meeting 16 Feb 2022 (#1170)
nodejs/TSC: meetings/2022-02-16.md

# Node.js Technical Steering Committee (TSC) Meeting 2022-02-16

## Links

* **Recording**: <https://www.youtube.com/watch?v=u6srlkRSRM4>
* **GitHub Issue**: <https://github.com/nodejs/TSC/issues/1166>

## Present

* Antoine du Hamel @aduh95 (TSC)
* Matteo Collina @mcollina (TSC)
* Tobias Nießen @tniessen (TSC)
* Gireesh Punathil @gireeshpunathil (TSC)
* Darshan Sen @RaisinTen (TSC)
* Beth Griggs @BethGriggs (TSC)
* Сковорода Никита Андреевич @ChALkeR (TSC)
* Richard Lau @richardlau (TSC)
* Michaël Zasso @targos (TSC)

## Agenda

### Announcements

> * The Node.js copyright has been donated by Joyent/Samsung to the OpenJS Foundation <https://openjsf.org/blog/2022/02/14/node-js-trademarks-transferred-to-openjs-foundation/>

Reviewer: ```suggestion
* The Node.js trademarks have been donated by Joyent/Samsung to the OpenJS Foundation <https://openjsf.org/blog/2022/02/14/node-js-trademarks-transferred-to-openjs-foundation/>
```

* OpenJS World CFP extended to 25th of February
* New collaborator!

### CPC and Board Meeting Updates

*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/node

* crypto: clarify `require("crypto").getRandomValues` is Node.js specific [#41782](https://github.com/nodejs/node/pull/41782)

>   * matteo: What is the problem? The question is unclear

Author: ```suggestion
  * Matteo: What is the problem? The question is unclear
```
